Tests/TemplatesTests/ColorsTests.swift

```diff
@@ -45,4 +45,18 @@ class ColorsTests: XCTestCase {
          directory: .colors,
          contextVariations: variations)
   }
+
+  func testLiteralsSwift3() {
+    test(template: "literals-swift3",
+         contextNames: Contexts.all,
+         directory: .colors,
+         contextVariations: variations)
```

Comment:
Shouldn't we also test the `--param enumName=UIColor` case too, to check the enum isn't created there?